### DISCUSSION TITLE
How can I disable fancyindex?

### DISCUSSION THREAD

**Discussion by hopoduck** [ZoeyVid/NPMplus#658] [Q&A] [answered]
I want to disable fancyindex and show 404 like before when I request access to the folder.

Is there a flag to disable this or set it as an environment variable?

This is my proxy host’s `Custom Nginx Configuration`

```
location / {
    root /var/www;
} 
```

**Comment by Zoey2936** [comment, marked as answer]
try `fancyindex off;`
